staging/coverage_config_x86_64.json

```diff
@@ -1,5 +1,5 @@
 {
-  "coverage_score": 57.35,
+  "coverage_score": 56.58,
```

Comment:
This is lower because we merged RX support without tests, and we didn't update it, right?